projects/cashmere/src/lib/pipes/age-pipe/age.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Date } from 'sugar';
+
+@Pipe({
+  'name': 'age'
+})
+export class AgePipe implements PipeTransform {
+
+  public transform(value: string): string {
+    const birth: Date = Date.create(value);
+
+    // if under 12 months then show in months
+    let ageInMonths: number = Date.monthsAgo(birth);
+    if (ageInMonths < 12) {
+      return `${ageInMonths} ` + ((ageInMonths === 0 || ageInMonths > 1) ? 'mos' : 'mo');
```

Comment:
We want Cashmere to work with localization out of the box.  How would you handle alternative locales?

Reply:
I honestly have no clue as I've never really done localization before.... what would you suggest?

Reply:
Take a look at how Angular handles it in the DatePipe: https://github.com/angular/angular/blob/fa699f65d7a2cb4d27b508e816acf256d03ec259/packages/common/src/pipes/date_pipe.ts#L156